src/less/field/field.less

```diff
@@ -148,3 +148,7 @@ div.field__description {
     align-self: start;
     margin-top: 16px;
 }
+
+input:disabled + label.field__label {
```

Comment:
Is this `field__label` applicable to the `field` component? There doesn't seem to be any markup changes in the component utilizing it.